bindings/rust/s2n-tls/src/security.rs

```diff
@@ -108,12 +108,15 @@ pub const DEFAULT_TLS13: Policy = policy!("default_tls13");
 #[cfg(feature = "pq")]
 pub const TESTING_PQ: Policy = policy!("PQ-TLS-1-0-2021-05-26");
 
+pub(crate) const TESTING_TLS12: Policy = policy!("20240501");
+
```

Comment:
Adding this didn't help as much as I thought it would :/ You've still got the integration tests and the tokio tests using the magic number. Maybe a `pub` policy is worth it.

Reply:
I think we should think hard before exposing more "default" policies, especially from the Rust bindings. For the moment, I have created a `TESTING_TLS12` policy gated to the `unstable-testing` feature. I consider this a 2-way decision so we can always change it if we need.